Source/WebCore/animation/WebAnimation.idl

```diff
@@ -50,8 +54,13 @@ typedef unsigned long FramesPerSecond;
     attribute DOMString id;
     [ImplementedAs=bindingsEffect] attribute AnimationEffect? effect;
     [EnabledConditionallyReadWriteBySetting=WebAnimationsMutableTimelinesEnabled] attribute AnimationTimeline? timeline;
+#if defined(ENABLE_CSS_TYPED_OM) && ENABLE_CSS_TYPED_OM
```

Comment:
We should probably kill this build time flag sooner rather than later. It is starting to make the code look pretty ugly and we have a runtime flag already.

Reply:
Yes, I will file a patch to do just that. Hopefully nothing bad happens.